segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperLog.java

```diff
@@ -635,18 +655,27 @@ private LogMetadata loadMetadata() throws DataLogInitializationException {
     /**
      * Updates the metadata and persists it as a result of adding a new Ledger.
      *
-     * @param currentMetadata The current metadata.
-     * @param newLedger       The newly added Ledger.
+     * @param currentMetadata   The current metadata.
+     * @param newLedger         The newly added Ledger.
+     * @param clearEmptyLedgers If true, the new metadata will not not contain any pointers to empty Ledgers. Setting this
+     *                          to true will not remove a pointer to the last few ledgers in the Log (controlled by
```

Comment:
There is one parenthesis that isn't closing.

Reply:
fixed

Reply:
ok.